### PR TITLE
Add python dependencies for import-idb

### PR DESCRIPTION
**Pull request by dwarfyresearch**
Needed for https://github.com/revng/revng/pull/199.

### CODE REVIEW

File: .orchestra/ci/requirements.txt [hunk 1]
```diff
@@ -44,6 +44,7 @@ pefile
 psutil
 pydot
 pyelftools
+python-idb
```
Comment (aleclearmind) [inline .orchestra/ci/requirements.txt]: Shouldn't we point this to our fork?

Comment (dwarfyresearch) [reply .orchestra/ci/requirements.txt]: Yes.

Comment (aleclearmind) [reply .orchestra/ci/requirements.txt]: OK, then you probably need to put the [URL of the git repo](https://www.activestate.com/resources/quick-reads/pip-install-git/). I guess your comment implied you knew that.

Comment (dwarfyresearch) [reply .orchestra/ci/requirements.txt]: But I already did it with: https://github.com/revng/orchestra/pull/95/commits/1f20a4893e1ab8d8ff1d23ed2de64a74f3ffe607. Is this enough?

Comment (aleclearmind) [reply .orchestra/ci/requirements.txt]: Ah OK, didn't notice. Mark as solved if done. Thanks! `:)`

**Pull request closed without merging**